README.md

```diff
@@ -1,156 +1,48 @@
-
-The confusion matrix diff tool shows tables of the largest score changes for
-the same examples, segmented according to the different sections of the
-confusion matrix. It shows the "new" false positives/negatives and true
-positives/negatives that one would get if upgrading from one model to the
-other.
-
-The score scatterplot tool plots the model's scores as a scatterplot. Each
-point represents an example, the original model's score is the `x` position and
-the new model's score is the `y` position. The points are colored according to
-the true label. For similar models, most points should fall close to the `y=x`
-line. Points that are far from that line are examples with larger score
-differences between the two models.
-
-If the new model on the y-axis is a proposed update to the model on the x-axis,
-then one would hope to see mostly positive labels in the upper left corner (new
-true positives) and mostly negative labels in the bottom right corner (new true
-negatives).
-
-## Data Description
-
-The `Prep Wikipedia Data.ipynb` notebook will generate the following datasets where
-SPLIT indicates whether the data is in the train, test, or dev splits:
-
-wiki_SPLIT.csv: The original Wikipedia data from the Figshare dataset, processed and split.
-wiki_debias_SPLIT.csv: The above data which is additionally augmented with Wikipedia article
-comments to debias on a set of terms (see `Dataset_bias_analysis.ipynb` for details).
-wiki_debias_random_SPLIT.csv: The wiki_SPLIT.csv augmented with a random selection of Wikipedia article
-columns that are of roughly the same length as those used to augment wiki_debias_SPLIT.csv. This is
-used as a control in our experiments.
+## Background
+
+As part of the Perspective API model training process, we evaluate identity-term
+bias in our models on synthetically generated and “templated” test sets where a
+range of identity terms are swapped into both toxic and non-toxic template
+sentences. For example, given templates like “I am a proud [identity] person”,
+we evaluate differences in score on sentences like:
+
+> “I am a proud Latino person”
+>
+> “I am a proud gay person”
+>
+> “I am a proud Muslim person”
+
+_Note that this evaluation looks at only the identity terms present in the
+text._
+
+For more reading on unintended bias and how we measure bias using the resulting
+model scores, see:
+
+- Our [overview](https://conversationai.github.io/bias.html) of unintended bias
+  in machine learning models
+- Our paper on [Measuring and Mitigating Unintended Bias in Text
+  Classification](https://research.google/pubs/pub46743/) for a deeper dive into
+  this approach and the metrics we use
+- Our [model
+  cards](https://developers.perspectiveapi.com/s/about-the-api-model-cards) for
+  an overview of our model training process and model performance metrics
+
+# Usage
+
+Sentence Templates is not itself sufficient for eliminating identity bias in
```

Comment:
I think
Sentence templates alone are insufficient for eliminating identity bias